recipe/meta.yaml

```diff
@@ -49,25 +49,25 @@ requirements:
     - menuinst >=1.4.11,<2         # [win]
   run:
     - python
-    - conda-package-handling >=1.3.0
+    - conda-package-handling >=2.2.0
     - menuinst >=1.4.11,<2         # [win]
     - packaging >=23.0
     - pycosat >=0.6.3
     - pyopenssl >=16.2.0
     - requests >=2.27.0,<3
     - ruamel.yaml >=0.11.14,<0.18
     - setuptools >=60.0.0
-    - toolz >=0.8.1
     - pluggy >=1.0.0
     - tqdm >=4
     - boltons >=23.0.0
     - jsonpatch >=1.32
+    - truststore >=0.8.0           # [py>=310]
   run_constrained:
     - conda-build >=3.18.3
     - conda-content-trust >=0.1.1
     - conda-env >=2.6
     - cytoolz >=0.8.1
```

Comment:
```suggestion
```
To match https://github.com/conda/conda/compare/23.7.4...23.9.0#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aL56